Flaky: `TestBakeCallCheckFlag`

since #2556

https://github.com/docker/buildx/actions/runs/9781672942/job/27006277267#step:7:422

```
=== FAIL: tests TestIntegration/TestBakeCallCheckFlag/worker=docker-container (2.04s)
    bake.go:1086: 
        	Error Trace:	/src/tests/bake.go:1086
        	            				/src/vendor/github.com/moby/buildkit/util/testutil/integration/run.go:96
        	            				/src/vendor/github.com/moby/buildkit/util/testutil/integration/run.go:211
        	Error:      	Received unexpected error:
        	            	invalid character '/' after top-level value
        	Test:       	TestIntegration/TestBakeCallCheckFlag/worker=docker-container
    --- FAIL: TestIntegration/TestBakeCallCheckFlag/worker=docker-container (2.04s)
```

--- a/tests/bake.go
+++ b/tests/bake.go
@@ -1074,15 +1074,16 @@ target "another" {
 	require.Contains(t, out, "another")
 	require.Contains(t, out, "UndefinedVar")
 
-	out, err = bakeCmd(
+	cmd := buildxCmd(
 		sb,
 		withDir(dir),
-		withArgs("build", "another", "--call", "check,format=json"),
+		withArgs("bake", "--progress=quiet", "build", "another", "--call", "check,format=json"),
 	)
-	require.Error(t, err, out)
+	outB, err := cmd.Output()
+	require.Error(t, err, string(outB))
 
 	var res map[string]any
-	err = json.Unmarshal([]byte(out), &res)
+	err = json.Unmarshal(outB, &res)
 	require.NoError(t, err, out)
 
 	targets, ok := res["target"].(map[string]any)

